packages/core/src/browser/tree/tree-expansion.ts

```diff
@@ -65,15 +67,16 @@ export interface ExpandableTreeNode extends CompositeTreeNode {
 }
 
 export namespace ExpandableTreeNode {
-    export function is(node: Object | undefined): node is ExpandableTreeNode {
-        return !!node && CompositeTreeNode.is(node) && 'expanded' in node;
+    export function is(node?: any): node is ExpandableTreeNode {
```

Comment:
Can you clarify why we want to now accept `any` given we try to avoid using it project wide?
Why for example in `ExpandableTreeNode` are we not doing like `SelectableTreeNode` and not accepting the parent `TreeNode` as a type?

Reply:
Because it doesn't really matter. The whole point of those functions is to tell TypeScript how to coerce values into specific types: `node is ExpandableTreeNode`.